### PR TITLE
Updated rails version 5.0.0.1 to 6.1.0

### PR DESCRIPTION
**Pull request by gonuguntla**

### CODE REVIEW

File: config/initializers/paper_trail.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # frozen_string_literal: true
 
-PaperTrail.config.track_associations = false
\ No newline at end of file
+# PaperTrail.config.track_associations = false
```
Comment (thisisstephenbetts) [inline config/initializers/paper_trail.rb:3]: Why is this commented out? Should we just remove this file?

File: test/functional/books_controller_test.rb [hunk 2]
```diff
@@ -21,7 +21,7 @@ class BooksControllerTest < ActionController::TestCase
   end
 
   test "test book strong params" do
-    post :create, { id: 1}
+    post :create, params: { call_number: "call_number" }
```
Comment (thisisstephenbetts) [inline test/functional/books_controller_test.rb:24]: Why did this change from using `id` to `call_number`?

(At some point this test could be improved by adding some more detailed assertions, but fine for now.)

**Review by thisisstephenbetts — APPROVED** [review]
Looks good! Nice work!

File: config/initializers/paper_trail.rb (hunk 1 above)
Comment (gonuguntla) [reply config/initializers/paper_trail.rb:3]: Ok Stephen

File: test/functional/books_controller_test.rb (hunk 2 above)
Comment (gonuguntla) [reply test/functional/books_controller_test.rb:24]: Just created book with another params instead of id.

Comment (thisisstephenbetts) [reply test/functional/books_controller_test.rb:24]: Oh okay — sounds good. Thanks!

**Pull request merged**